filebeat/docs/command-line.asciidoc

```diff
@@ -1,9 +1,15 @@
 [[filebeat-command-line]]
 === Command Line Options
 
-Filebeat does not have any Filebeat-specific command line options. Instead, you
+Filebeat does only have one Filebeat-specific command line option. For all other options you
```

Comment:
I would say, "The following command line option is specific to Filebeat." I'd just remove "For all other options you configure the behaviour of Filebeat by specifying options in the configuration file." 

BTW, I'm thinking about reworking the CLI topics to use conditional tagging instead of includes (that way we could just list all the flags and not explain where they are coming from).